.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ext/opcache/jit/ir"]
+	path = ext/opcache/jit/ir
+	url = https://github.com/dstogov/ir.git
```

Comment:
I also not a fun of submodules.
In general it's possible to embed IR into php-src, but this would complicate the development workflow (similar to libdate embedded into ext/date).

May be anyone may propose alternative solutions? (I already tried "git subtree" and "git subproject")

Reply:
Instead of embedding IR into php-src itself, it can be forked by [php](https://github.com/php) and kept as a separate repository. 
However, this would still require the use of git submodules

Reply:
I don't hate submodules but I think they'd complicate downstream packaging at the very least. If `ir` must be a separate repo I'd vote for the timelib pattern.

Reply:
I've replaced submodule with embedded version.